### ISSUE TITLE
Lib didn't work with tables with ltree column (postgresql)

### ISSUE THREAD

**Issue by alexesDev** [sdispater/orator#59]
I use ltree extension and catch error...
http://www.postgresql.org/docs/9.4/static/ltree.html

```
Traceback (most recent call last):
  File "main.py", line 32, in <module>
    print(Task.pending().get()[0].document.items[0].product.name)
  File "/usr/lib/python2.7/site-packages/orator/orm/utils.py", line 184, in __get__
    relation = self._get(instance)
  File "/usr/lib/python2.7/site-packages/orator/orm/utils.py", line 290, in _get
    _wrapped=False
  File "/usr/lib/python2.7/site-packages/orator/orm/model.py", line 776, in belongs_to
    instance = self._get_related(related)()
  File "/usr/lib/python2.7/site-packages/orator/orm/model.py", line 121, in __init__
    self._boot_if_not_booted()
  File "/usr/lib/python2.7/site-packages/orator/orm/model.py", line 148, in _boot_if_not_booted
    klass._boot()
  File "/usr/lib/python2.7/site-packages/orator/orm/model.py", line 167, in _boot
    cls._boot_columns()
  File "/usr/lib/python2.7/site-packages/orator/orm/model.py", line 172, in _boot_columns
    columns = connection.get_schema_manager().list_table_columns(cls.__table__ or inflection.tableize(cls.__name__))
  File "/usr/lib/python2.7/site-packages/orator/dbal/schema_manager.py", line 33, in list_table_columns
    return self._get_portable_table_columns_list(table, table_columns)
  File "/usr/lib/python2.7/site-packages/orator/dbal/schema_manager.py", line 66, in _get_portable_table_columns_list
    column = self._get_portable_table_column_definition(table_column)
  File "/usr/lib/python2.7/site-packages/orator/dbal/postgres_schema_manager.py", line 51, in _get_portable_table_column_definition
    type = self._platform.get_type_mapping(db_type)
  File "/usr/lib/python2.7/site-packages/orator/dbal/platforms/platform.py", line 359, in get_type_mapping
    return self.INTERNAL_TYPE_MAPPING[db_type]
KeyError: u'ltree'
```

**Comment by alexesDev** [comment]
Quick fix:
```python
db = DatabaseManager(config)
platform = db.get_database_platform()
platform.INTERNAL_TYPE_MAPPING['ltree'] = 'string'
```

**Comment by sdispater** [comment]
Thanks for reporting this.

It is a known issue in models that has been fixed in the `develop` branch (see 6c3d4a17189686f7bfc86f27904800fb84647ce2).

So, you can either use this branch or wait for the `0.8.1` version that should be released this week.

**Comment by alexesDev** [comment]
Same problem for column with int[] type.
